NPE in NoteListFragment

```
java.lang.RuntimeException: An error occured while executing doInBackground()
at android.os.AsyncTask$3.done(AsyncTask.java:300)
at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
at java.util.concurrent.FutureTask.run(FutureTask.java:242)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'com.automattic.simplenote.utils.TagsAdapter$TagMenuItem com.automattic.simplenote.NotesActivity.getSelectedTag()' on a null object reference
at com.automattic.simplenote.NoteListFragment.queryNotes(NoteListFragment.java:457)
at com.automattic.simplenote.NoteListFragment$refreshListTask.doInBackground(NoteListFragment.java:708)
at com.automattic.simplenote.NoteListFragment$refreshListTask.doInBackground(NoteListFragment.java:702)
at android.os.AsyncTask$2.call(AsyncTask.java:288)
at java.util.concurrent.FutureTask.run(FutureTask.java:237)
... 3 more
```

--- a/Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java
+++ b/Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java
@@ -452,7 +452,7 @@ public class NoteListFragment extends ListFragment implements AdapterView.OnItem
         refreshList(true);
     }
 
-    public ObjectCursor<Note> queryNotes(){
+    public ObjectCursor<Note> queryNotes() {
         NotesActivity notesActivity = (NotesActivity)getActivity();
         Query<Note> query = notesActivity.getSelectedTag().query();
 

--- a/Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java
+++ b/Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java
@@ -414,6 +414,10 @@ public class NotesActivity extends ActionBarActivity implements
     }
 
     public TagsAdapter.TagMenuItem getSelectedTag() {
+        if (mSelectedTag == null) {
+            mSelectedTag = mTagsAdapter.getDefaultItem();
+        }
+        
         return mSelectedTag;
     }
 

